azurelinuxagent/common/utils/networkutil.py

```diff
@@ -160,34 +160,35 @@ def __get_common_command_params(command, destination):
         return ["-t", "security", command, "OUTPUT", "-d", destination, "-p", "tcp", "-m"]
 
     @staticmethod
-    def get_accept_tcp_rule(command, destination, firewalld_command="", wait=""):
-        # This rule allows DNS TCP request to wireserver ip for non root users
-
+    def __get_firewall_base_command(firewalld_command="", wait=""):
+        # Firewalld.service fails if we set `-w` in the iptables command, so not adding it at all for firewalld commands
         if firewalld_command != "":
             cmd = AddFirewallRules.__get_firewalld_base_command(firewalld_command)
         else:
             cmd = AddFirewallRules.__get_iptables_base_command(wait)
+
+        return cmd
+
+    @staticmethod
+    def get_accept_tcp_rule(command, destination, firewalld_command="", wait=""):
+        # This rule allows DNS TCP request to wireserver ip for non root users
+        cmd = AddFirewallRules.__get_firewall_base_command(firewalld_command, wait)
+
         cmd = cmd + ['-t', 'security', command, 'OUTPUT', '-d', destination, '-p', 'tcp', '--destination-port', '53',
                      '-j', 'ACCEPT']
         return cmd
 
     @staticmethod
-    def get_accept_command(command, destination, owner_uid, firewalld_command="", wait=""):
-        if firewalld_command != "":
-            cmd = AddFirewallRules.__get_firewalld_base_command(firewalld_command)
-        else:
-            cmd = AddFirewallRules.__get_iptables_base_command(wait)
+    def get_wire_root_accept_rule(command, destination, owner_uid, firewalld_command="", wait=""):
+        cmd = AddFirewallRules.__get_firewall_base_command(firewalld_command, wait)
 
         cmd.extend(AddFirewallRules.__get_common_command_params(command, destination))
```

Comment:
The TCP rule uses - `cmd + [-t, ..]` notations and the other rules use `cmd.extend([...])`, maybe make them both consistent for better readability?